package.json

```diff
@@ -28,6 +28,7 @@
     "cookies": "0.7.3",
     "cors": "2.8.5",
     "dayjs": "1.8.14",
+    "envinfo": "7.3.1",
```

Comment:
Nice, much better, but need to include the `yarn.lock` file updated, otherwise the merge will fails. To achieve that, just run `yarn` in your local folder and commit the lock file.